src/Symfony/Component/AssetMapper/Compressor/ChainCompressor.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AssetMapper\Compressor;
+
+/**
+ * Calls multiple compressors in a chain.
+ *
+ * @author Kévin Dunglas <kevin@dunglas.dev>
+ */
+final readonly class ChainCompressor implements CompressorInterface
+{
+    /**
+     * @param CompressorInterface[] $compressors
+     */
+    public function __construct(
+        private array $compressors = [
+            new BrotliCompressor(),
+            new ZstandardCompressor(),
+            new GzipCompressor(),
```

Comment:
shouldn't the default value be to enable only supported compressors ?

Reply:
done